github/Organization.py

```diff
@@ -462,30 +401,31 @@ def create_hook(
         """
         assert isinstance(name, str), name
         assert isinstance(config, dict), config
-        assert events is github.GithubObject.NotSet or all(isinstance(element, str) for element in events), events
-        assert active is github.GithubObject.NotSet or isinstance(active, bool), active
-        post_parameters = {
+        assert is_optional_list(events, str), events
+        assert is_optional(active, bool), active
+        post_parameters: dict[str, Any] = {
```

Comment:
why not having `NotSet.remove_unset_items` with `events` and `active` combined into this dict declaration?